libs/utils/src/auth.ts

```diff
@@ -0,0 +1,45 @@
+import { DippedSmartCardAuth, InsertedSmartCardAuth } from '@votingworks/types';
```

Comment:
How did you choose `libs/utils` vs `libs/types` for these? Maybe doesn't super matter since often both libs are used together, but seems like type guards could live with the associated types

Reply:
Yeah good q! I was torn between these two options. I went with `libs/utils` following the simple rule that `libs/types` should only contain types (though I know that's not entirely the case even today)

I do like the idea of colocating logic though, so I think I'll move the type guards to `libs/types` after all

Reply:
Ah! Actually ran into a hiccup here. The tests for the type guards use `@votingworks/test-utils`. Moving the type guards and their corresponding tests to `@votingworks/types` would create a circular dependency between `@votingworks/test-utils` and `@votingworks/types`. I could of course just not use the test utils, but I think the pros of keeping the type guards in `@votingworks/utils` now outweigh the pros of moving them to `@votingworks/types`

Reply:
Classic dependency cycle hiccup. Sounds good